x-pack/plugins/canvas/public/components/workpad_header/workpad_header.js

```diff
@@ -59,12 +58,7 @@ export class WorkpadHeader extends React.PureComponent {
           maxWidth="1000px"
           initialFocus=".canvasElements__filter"
         >
-          <ElementTypes
-            onClick={element => {
-              addElement(element);
-              setShowElementModal(false);
-            }}
-          />
+          <ElementTypes onClose={() => setShowElementModal(false)} />
```

Comment:
Be aware this function is recreated, and might cause a re-render, each time... depending on how it's used.